sentry_sdk/integrations/aws_lambda.py

```diff
@@ -93,16 +89,14 @@ def sentry_handler(aws_event, aws_context, *args, **kwargs):
             # this is empty
             request_data = {}
 
-        hub = Hub.current
-        integration = hub.get_integration(AwsLambdaIntegration)
+        client = sentry_sdk.get_client()
+        integration = client.get_integration(AwsLambdaIntegration)
         if integration is None:
             return handler(aws_event, aws_context, *args, **kwargs)
 
-        # If an integration is there, a client has to be there.
-        client = hub.client  # type: Any
         configured_time = aws_context.get_remaining_time_in_millis()
 
-        with hub.push_scope() as scope:
+        with sentry_sdk.new_scope() as scope:
```

Comment:
`push_scope` forks the isolation scope, not the current scope like `new_scope`. Are we sure this is the desired behavior here? Further, should `push_scope` perhaps be forking the current scope instead of the isolation scope? cc @antonpirker